prow/plugins/approve/approve.go

```diff
@@ -125,12 +125,32 @@ func helpProvider(config *plugins.Configuration, enabledRepos []string) (*plugin
 		}
 		approveConfig[repo] = fmt.Sprintf("Pull requests %s require an associated issue.<br>Pull request authors %s implicitly approve their own PRs.<br>The /lgtm [cancel] command(s) %s act as approval.<br>A GitHub approved or changes requested review %s act as approval or cancel respectively.", doNot(opts.IssueRequired), doNot(opts.HasSelfApproval()), willNot(opts.LgtmActsAsApprove), willNot(opts.ConsiderReviewState()))
 	}
+
+	yamlSnippet, err := plugins.CommentMap.GenYaml(&plugins.Configuration{
```

Comment:
What about trying to avoid the golang marshalling issue by not rendering the help for a `*plugins.Configuration` but just for a `[]plugins.Approve`?

Reply:
@alvaroaleman - the one major drawback with that is that the top-level plugin *key* will be excluded from the result (which may be confusing):

```diff
- # Built-in plugins specific configuration.
- approve:
```